packages/aragon-cli/src/commands/apm_cmds/util/acl.js

```diff
@@ -3,18 +3,27 @@ const { getRecommendedGasLimit } = require('../../../util')
 module.exports = ({ web3, gasPrice, network }) => {
   const getACL = async repoAddr => {
     const repo = new web3.eth.Contract(
-      require('../abi/os/AragonApp').abi,
+      require('@aragon/os/build/contracts/AragonApp').abi,
```

Comment:
This syntax definitely works, but I saw a different syntax in other aragon packages. e.g.: https://github.com/aragon/apm.js/blob/master/src/index.js#L159
@sohkai Do you have any recommendation for one or the other?